TestAssets/TestProjects/KestrelSample/KestrelStandalone/project.json

```diff
@@ -12,9 +12,9 @@
     "../src/*.cs"
   ],
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
       "dependencies": {
-        "Microsoft.NETCore.App": "1.0.0-rc2-24008"
+        "Microsoft.NETCore.App": "1.0.0-rc2-3002306"
```

Comment:
After generating the stub package for x86/x64 on windows we can use the *.

Will make this change in a separate PR.